meilisearch-release-changelog/v1.15.0.md

```diff
@@ -0,0 +1,63 @@
+# v1.15.0 release changelogs
+
+<!-- The following line should ONLY be put PRE-release changelogs -->
+> [!WARNING]
+> Since this is a release candidate (RC), we do NOT recommend using it in a production environment. Is something not working as expected? We welcome [bug reports](https://github.com/meilisearch/meilisearch/issues/new/choose) and [feedback about new features](https://github.com/meilisearch/product/discussions).
+
+Meilisearch v1.15.0 introduces improves the indexing performances and dumpless experience.
+
+<!-- The following lines should NOT be put in the PRE-release changelogs -->
+🧰 All official Meilisearch integrations (including SDKs, clients, and other tools) are compatible with this Meilisearch release. Integration deployment happens between 4 to 48 hours after a new version becomes available.
+
+<!-- The following lines should NOT be put in the PRE-release changelogs -->
+Some SDKs might not include all new features. Consult the project repository for detailed information. Is a feature you need missing from your chosen SDK? Create an issue letting us know you need it, or, for open-source karma points, open a PR implementing it (we'll love you for that ❤️).
+
+# New features and updates 🔥
+
+## Deactivate Typo Tolerance on Numbers
+
+TBD
+
+Done by @ManyTheFish in [#5494](https://github.com/meilisearch/meilisearch/pull/5494).
+
+## Make it Possible to Rollback Dumpless Upgrades
+
+TBD
+
+Done by @dureuill in [#5523](https://github.com/meilisearch/meilisearch/pull/5523).
+
+## Other improvements
+
+* Enhance filterable error messages by @CodeMan62 in [#5425](https://github.com/meilisearch/meilisearch/pull/5425)
+* Improve error messages on embeddings dimension mismatch by @vuthanhtung2412 in [#5449](https://github.com/meilisearch/meilisearch/pull/5449)
+* Support EC private key as SSL certificate by @HDT3213 in [#5471](https://github.com/meilisearch/meilisearch/pull/5471)
+* Stop compacting the snapshots to speedup their generation by @dureuill in [#5498](https://github.com/meilisearch/meilisearch/pull/5498)
+* Update network URL validation error message format to match expected by @CodeMan62 in [#5486](https://github.com/meilisearch/meilisearch/pull/5486)
+* Add a new `batchCreationComplete` field in the batches stats by @dureuill in [#5488](https://github.com/meilisearch/meilisearch/pull/5488) and [#5530](https://github.com/meilisearch/meilisearch/pull/5530)
+* Expose the size of the status of the task queue in the prometheus metrics by @DanasFi in [#5512](https://github.com/meilisearch/meilisearch/pull/5512)
```

Comment:
Can we link the associated experimental feature?
https://github.com/orgs/meilisearch/discussions/625